tests/contrib/actions/email/email/test_email.py

```diff
@@ -0,0 +1,116 @@
+import pytest
+from scrapy.utils.test import get_crawler
+
+from spidermon.contrib.actions.email import SendEmail
+
+
+class SendEmailTest(SendEmail):
+    """A helper class to override the base class behavior of formatting templates.
+    This way no setup is required for a simple test."""
+
+    def get_subject(self):
+        return self.subject
+
+    def get_body_text(self):
+        return self.body_text
+
+    def get_body_html(self):
+        return self.body_html
+
+
+@pytest.mark.parametrize(
```

Comment:
Hey @rennerocha , updated with this approach.
However, I think there is some code duplication and I don't think that it would nice to extract it as a function.